### PR TITLE
Kevin Pho dev teams pages fixes

### PR DESCRIPTION
**Pull request by kevinpho2022**
https://github.com/SSWConsulting/SSW.Website.Content/issues/19

Added required props for enabling changes made in SSW.Website.content
Added props to react template
Added conditional to handle if there's nothing in the technology list

### CODE REVIEW

File: src/styles/global.css [hunk 1]
```diff
@@ -34,9 +34,7 @@ p {
 }
 
 a {
-    color: var(--grey);
-    border-bottom: 1px dotted #9e9e9e;
-    text-decoration: none;
+    color:white;
```
Comment (lukecookssw) [inline src/styles/global.css]: Move this styling out of global.css and into the component's specific CSS file. We don't want it to apply to every anchor tag across the entire website.

File: src/templates/consulting/components/benefits/index.tsx [hunk 2]
```diff
@@ -41,6 +41,10 @@ const Benefits = ({ benefits, markdownContent }) => {
                                 <h4 className={styles.benefitTitle}>{title}</h4>
                                 <article className={styles.benefitDesc}>
                                     {description}
+                                    {' '}
```
Comment (lukecookssw) [inline src/templates/consulting/components/benefits/index.tsx]: Remove this hard-coded space and instead use CSS styling such as margin-left on the anchor tag.

File: src/templates/consulting/components/technologies/index.tsx [hunk 3]
```diff
@@ -14,9 +14,14 @@ import NodeJsTech from '../../../../components/technologies/node-js';
...
-        techList.map(({name}, index) => (
-            setTechComponents(techComponents => [...techComponents, getComponent(name)])
-        ));
+        if (techList) {
+            techList.map(({name}, index) => (
+                setTechComponents(techComponents => [...techComponents, getComponent(name)])
+            ));
+        } else {
+            return (null)
+        }
```
Comment (lukecookssw) [inline src/templates/consulting/components/technologies/index.tsx]: Is the else statement required? I don't think it is.

File: src/templates/consulting/components/technologies/index.tsx [hunk 4]
```diff
@@ -39,122 +44,21 @@ const Technologies = ({ techList }) => {
...
-                </div> */}
                 </div>
             </article>
         </section>
-    )
+        )  
+    } else {
+        return (null)
+    }
+
```
Comment (lukecookssw) [inline src/templates/consulting/components/technologies/index.tsx]: Remove the else statement and instead do:

if (techList &&)

(the syntax above is pseudo-code, but pretty close. There's a weird thing in React where it forces you to put an else statement _unless_ you use this && operator.

**Review by lukecookssw — CHANGES_REQUESTED** [review]
Please see changes noted.

File: src/templates/consulting/components/benefits/index.tsx (hunk 2 above)
Comment (kevinpho2022) [reply src/templates/consulting/components/benefits/index.tsx]: used margin-left as recommended

File: src/styles/global.css (hunk 1 above)
Comment (kevinpho2022) [reply src/styles/global.css]: moved into component specific css file

File: src/templates/consulting/components/technologies/index.tsx (hunk 3 above)
Comment (kevinpho2022) [reply src/templates/consulting/components/technologies/index.tsx]: removed

File: src/templates/consulting/components/technologies/index.tsx (hunk 4 above)
Comment (kevinpho2022) [reply src/templates/consulting/components/technologies/index.tsx]: removed the else clause

**Review by lukecookssw — APPROVED** [review]
LGTM

**Pull request merged**